pkg/projects/Microsoft.NETCore.DotNetHost/Microsoft.NETCore.DotNetHost.pkgproj

```diff
@@ -4,70 +4,9 @@
 
   <PropertyGroup>
     <Version>$(HostVersion)</Version>
-    <SkipPackageFileCheck>true</SkipPackageFileCheck>
-    <SkipValidatePackage>true</SkipValidatePackage>
-    <PackagePlatforms>x64;x86;arm;arm64;armel;</PackagePlatforms>
-    <OutputPath>$(PackagesOutputPath)</OutputPath>
-    <IncludeRuntimeJson>true</IncludeRuntimeJson>
-    <Serviceable>true</Serviceable>
```

Comment:
What Matt said was the intention, but you reminded me that Microsoft.NETCore and Microsoft.NETCore.UniversalWindowsPlatform  bypass importing that dir.props, so I moved it up a level into `pkg/dir.props`